pkg/ddevapp/webserver_config_assets/nginx-site-typo3.conf

```diff
@@ -5,6 +5,12 @@
 # and ddev will respect it and won't overwrite the file.
 # See https://ddev.readthedocs.io/en/stable/users/extend/customization-extendibility/#providing-custom-nginx-configuration
 
+# Support for WebP
+map $http_accept $webp_suffix {
```

Comment:
I don't know about that `webp` part here...

Reply:
This is used later (line 51) to enabled webp serving if the broesers states webp support. Not related to the main issue concering the CSP see also comment on lines 49-51.